go/libkb/errors.go

```diff
@@ -1399,6 +1399,12 @@ func (e UnknownStreamError) Error() string {
 	return "unknown stream format"
 }
 
+type UnicodeUnsupportedError struct{}
+
+func (e UnicodeUnsupportedError) Error() string {
+	return "unicode not supported"
```

Comment:
Since it's so Windows-specific, maybe we could even suggest a command that would convert the file back to ASCII/UTF-8? Though I could see that being more trouble than it's worth if the only way to do it depends on having powershell installed.